spec.html

```diff
@@ -890,6 +890,32 @@ <h1>AsyncContext.Snapshot.prototype</h1>
       </emu-clause>
     </emu-clause>
 
+    <emu-clause id="sec-function-properties-of-the-asynccontext-snapshot-object">
+      <h1>Function Properties of the AsyncContext.Snapshot Object</h1>
```

Comment:
Aren't static methods usually put in "properties of the X constructor"?

Reply:
Oh, I think this wording was due to it being on the `AsyncContext` namespace object, not a constructor. Updating.